docs/2.build/5.primitives/ft.md

```diff
@@ -70,9 +78,15 @@ On initialization you will define the token's metadata such as its name (e.g. Et
 
 To initialize a FT contract you will need to deploy it and then call the `new` method defining the token's metadata.
 
-```bash
-cargo near deploy build-non-reproducible-wasm <account-id> with-init-call new json-args '{"owner_id": "<owner-account>", "total_supply": "1000000000000000", "metadata": { "spec": "ft-1.0.0", "name": "Example Token Name", "symbol": "EXLT", "decimals": 8 }}' prepaid-gas '100.0 Tgas' attached-deposit '0 NEAR' network-config testnet sign-with-keychain send
-```
+<Tabs groupId="code-tabs">
+  <TabItem value="🖥️ Cargo-NEAR" label="🖥️ Cargo-NEAR">
+    <CargoNearCreateTokenManually />
```

Comment:
In this case I'd leave the `cargo` code here, like:

```suggestion

```bash
cargo near deploy build-non-reproducible-wasm <account-id> with-init-call new json-args '{"owner_id": "<owner-account>", "total_supply": "1000000000000000", "metadata": { "spec": "ft-1.0.0", "name": "Example Token Name", "symbol": "EXLT", "decimals": 8 }}' prepaid-gas '100.0 Tgas' attached-deposit '0 NEAR' network-config testnet sign-with-keychain send
```

```

Reply:
I would not do this because:
1. It breaks the general pattern where we use components inside the tabs. Mixing two approaches makes the documentation look inconsistent, making it harder for other contributors to understand when to use components and when to keep them in the file.
2. It complicates editing and reading because it eliminates spaces, making it difficult to recognize the nesting.

Reply:
@bucanero also I would recommend the format this command and improve the readability:

```bash
cargo near deploy build-non-reproducible-wasm <account-id> \
  with-init-call new \
  json-args '{
    "owner_id": "<owner-account>",
    "total_supply": "1000000000000000",
    "metadata": {
      "spec": "ft-1.0.0",
      "name": "Example Token Name",
      "symbol": "EXLT",
      "decimals": 8
    }
  }' \
  prepaid-gas '100.0 Tgas' \
  attached-deposit '0 NEAR' \
  network-config testnet \
  sign-with-keychain send
```

P.S We can use AI for such tasks.

Reply:
yes, it looks better for this `cargo near` command 👍